src/components/bulkConcepts/container/BulkConceptsPage.jsx

```diff
@@ -78,10 +78,12 @@ export class BulkConceptsPage extends Component {
     } = event;
     const targetName = name.split(',');
     const query = `q=${this.state.searchInput}`;
+    this.setState({ searchingOn: true });
+    const { currentPage } = this.props;
     if ((targetName[1]).trim() === 'datatype') {
-      this.props.addToFilterList(targetName[0], 'datatype', query);
+      this.props.addToFilterList(targetName[0], 'datatype', query, currentPage);
```

Comment:
Since `addToFilterList` is reused, Please add it to the destructuring on line `82`

Reply:
Destructuring the action `addToFilterList` on line `82` causes `no-shadow` ESlint error. There are workarounds that can be made as observed [here](https://stackoverflow.com/questions/37682705/avoid-no-shadow-eslint-error-with-mapdispatchtoprops) but they can introduce in some complexity like reducing readability, introducing in some boilerplate code and disabling eslint isn't ideal. By explicitly using the method off of the props object, we don't need to worry about shadowing.

Reply:
Alright